app/services/doc_auth/socure/requests/docv_result_request.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module DocAuth
+  module Socure
+    module Requests
+      class DocvResultRequest < DocAuth::Socure::Request
+        attr_reader :document_capture_session_uuid, :biometric_comparison_required
+
+        def initialize(document_capture_session_uuid:, biometric_comparison_required: false)
+          @document_capture_session_uuid = document_capture_session_uuid
+          @biometric_comparison_required = biometric_comparison_required
+        end
+
+        private
+
+        def body
+          {
+            modules: ['documentverification'],
+            docvTransactionToken: document_capture_session.socure_docv_transaction_token,
+            customerUserId: document_capture_session_uuid,
```

Comment:
```suggestion
```
after our chat with Socure this week, we are discontinuing our use of customerUserID